RecoEcal/EgammaClusterProducers/python/particleFlowSuperClusteringSequence_cff.py

```diff
@@ -8,7 +8,8 @@
 # Producer for energy corrections
 #from RecoEcal.EgammaClusterProducers.correctedDynamicHybridSuperClusters_cfi import *
 # PFECAL super clusters, either hybrid-clustering clone (Box) or mustache.
-particleFlowSuperClusteringSequence = cms.Sequence(particleFlowSuperClusterECAL)
+particleFlowSuperClusteringTask = cms.Task(particleFlowSuperClusterECAL)
+#particleFlowSuperClusteringSequence = cms.Sequence(particleFlowSuperClusterECAL)
```

Comment:
Either remove (preferred, as it is not needed) or uncomment (if you want to maintain a sequence with the same name as the file).
If uncommented, please use
```PYTHON
particleFlowSuperClusteringSequence = cms.Sequence(particleFlowSuperClusteringTask)
```